tests/github/webhooks_test.py

```diff
@@ -4,12 +4,22 @@
 
 from pathlib import Path
 
-from safir.github import webhooks
 from safir.github.models import (
     GitHubCheckRunStatus,
     GitHubCheckSuiteConclusion,
     GitHubCheckSuiteStatus,
 )
+from safir.github.webhooks import (
```

Comment:
Sorry about this; it was just because `GitHubAppInstallationRepositoriesEventModel` is otherwise long enough that one of the code lines was unavoidably over 80 characters.